lcls_tools/common/image/processing.py

```
@@ -405,25 +406,41 @@ def process_images(
 
     # center the images
     if center:
-        if image_centroids is None:
-            image_centroids = calc_image_centroids(images, image_fitter=image_fitter)
-        centered_images = center_images(images, image_centroids)
+        try:
+            if image_centroids is None:
+                image_centroids = calc_image_centroids(
+                    images, image_fitter=image_fitter
+                )
+            centered_images = center_images(images, image_centroids)
+        except ValueError:
```

This is fine for now since we're passing a value in, but in general we should throw a specific error for cases like this.

---

Is there a way to test for this independent of fit method?